### ISSUE TITLE
New REPL Traceback Not Formatted Correctly

### ISSUE THREAD

**Issue by Siecje** [python/cpython#123178]
# Bug report

### Bug description:

At first I thought it was because the window was small but it happens even if the window is large.

```Python
>>> import threading
>>> thread = threading.Thread(target=lambda: threading.current_thread().name == 'MainThread')
>>> thread.run()
>>> thread.join()
Traceback (most recent call last):
  File "<python-input-3>", line 1, in <module>
    thread.join()
    ~~~~~~~~~~~^^
  File "/Users/codyscott_1/.pyenv/versions/3.13t-dev/lib/python3.13t/threading.py", line 1083, in join
    raise RuntimeError("cannot join thread before it is started")
RuntimeError: cannot join thread before it is started
>>> thread.start()
Exception in thread Thread-1 (<lambda>):
>>> Traceback (most recent call last):
                                        File "/Users/codyscott_1/.pyenv/versions/3.13t-dev/lib/python3.13t/threading.py", line 991, in run
                                                                if self._target is not None:
                     ^^^^^^^^^^^^
                                 AttributeError: 'Thread' object has no attribute '_target'. Did you mean: '_started'?

                                        During handling of the above exception, another exception occurred:

                             Traceback (most recent call last):
                                                                 File "/Users/codyscott_1/.pyenv/versions/3.13t-dev/lib/python3.13t/threading.py", line 1041, in _bootstrap_inner
                         self.run()
                                       ~~~~~~~~^^
                                                   File "/Users/codyscott_1/.pyenv/versions/3.13t-dev/lib/python3.13t/threading.py", line 996, in run
                                                                           del self._target, self._args, self._kwargs
                                               ^^^^^^^^^^^^
                                                           AttributeError: 'Thread' object has no attribute '_target'

>>> 
```

### CPython versions tested on:

3.13

### Operating systems tested on:

macOS

**Comment by cfbolz** [comment]
confirmed on linux

**Comment by rruuaanng** [comment]
I tested it on Ubuntu 22 and didn't encounter the issues mentioned, Maybe it's because my computer isn't a MacOS
Python 3.13.0rc1+ (heads/3.13:e5568e5e9da, Sep  6 2024, 09:06:23) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
warning: can't use pyrepl: No module named 'msvcrt'
>>> import threading
>>> thread = threading.Thread(target=lambda: threading.current_thread().name == 'MainThread')
>>> thread.run()
>>> thread.join()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
    thread.join()
    ~~~~~~~~~~~^^
  File "/home/ruang/Desktop/cpython/Lib/threading.py", line 1083, in join
    raise RuntimeError("cannot join thread before it is started")
RuntimeError: cannot join thread before it is started
>>> thread.start()
Exception in thread Thread-1 (<lambda>):
>>> Traceback (most recent call last):
  File "/home/ruang/Desktop/cpython/Lib/threading.py", line 991, in run
    if self._target is not None:
       ^^^^^^^^^^^^
AttributeError: 'Thread' object has no attribute '_target'. Did you mean: '_started'?

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ruang/Desktop/cpython/Lib/threading.py", line 1041, in _bootstrap_inner
    self.run()
    ~~~~~~~~^^
  File "/home/ruang/Desktop/cpython/Lib/threading.py", line 996, in run
    del self._target, self._args, self._kwargs
        ^^^^^^^^^^^^
AttributeError: 'Thread' object has no attribute '_target'

**Comment by ambv** [comment]
Fixed by #123799.
